lib/evervault.d.ts

```diff
@@ -1,7 +1,8 @@
 declare module "@evervault/sdk" {
     export default class Evervault {
-        constructor(apiKey: string)
+        constructor(apiKey: string, appUuid: string)
         encrypt: (data: any) => Promise<any>;
+        decrypt: (encryptedData: any) => Promise<any>;
```

Comment:
Set the arg type as `Object || Buffer` and the result to `Promise<Object || Buffer>`